site/tr_TR.all.json

```diff
@@ -363,7 +363,7 @@
     "classification_outro": { "other": "" },
   
     "cookie_banner_message": { "other": "Bu site kullanıcı deneyimini artırmak için çerezler kullanmaktadır. Bu siteyi kullanırken, < çerez ve gizlilik ayarlarını kabul edin <a href=\"{{.CookiePolicyURL}}\">cookie usage</a>." },
-    "cookie_banner_accept": { "other": "Kabul Ediyorum" },
+    "cookie_banner_accept": { "other": "Kabul Etmek" },
```

Comment:
Doesn't this need to be one word to avoid the wrapping issue?

Reply:
I did have a thought/concern there could be a language where a single word wouldn't make sense, so we'd have to accomodate anyway. I'm not saying that is the case with this language (because how would we know!), just BTW.